src/dynarmic/interface/A32/a32.h

```diff
@@ -91,6 +92,12 @@ class Jit final {
     /// Debugging: Dump a disassembly all compiled code to the console.
     void DumpDisassembly() const;
 
+    /* 
```

Comment:
```suggestion
    /** 
```

In order for these to be picked up by documentation generators like Doxygen. Ditto for the a64 equivalent below